libs/utils/env.py

```diff
@@ -644,6 +644,17 @@ def _init_platform(self):
         # Adding topology information
         self.platform['topology'] = self.topology.get_level("cluster")
 
+        # Adding kernel build information
+        kver = self.target.execute('uname -r').strip()
```

Comment:
No need to change any existing APIs, the only function in `KernelVersion` right now is `__str__`/`__repr__` so we could just add properties.